tests/README.md

```diff
@@ -0,0 +1,53 @@
+# Tests #
+
+PHP Client for GridGain contains [PHPUnit](https://phpunit.de/) tests to check the behavior of the client. The tests include:
+- functional tests which cover all API methods of the client
+- examples executors which run all examples except AuthTlsExample
+- AuthTlsExample executor
+
+The client has been tested on the following platforms:
+- Ubuntu 14.04 LTS 32-bit
```

Comment:
I believe we should make sure it works on the latest LTS (18.04). 14 is way too old.

Reply:
Is there anything platform-specific anyway?
Because this OS list is very short and gives an impression of poorly tested product.

I think we can remove this list altogether, or mention that client works on Windows/Linux/MacOS as it is supposed to do.

Reply:
Agree. Removed the list altogether.